crates/logger/src/logger.rs

```diff
@@ -59,8 +66,33 @@ impl Logger {
                     message
                 ));
             })
-            .chain(Dispatch::new().level(level).chain(io::stderr()))
-            .apply()
-            .unwrap();
+            .chain(io::stdout());
```

Comment:
@milesj this used to be stderr. that felt weird to me but idk what i'm talking about with this shit

Reply:
Aww yeah, that was correct, I missed this.

Logs are written to stderr so they can be piped/captured separately from normal output.

Reply:
weird